alternatives.c

```diff
@@ -1304,6 +1330,10 @@ int main(int argc, const char **argv) {
     struct stat sb;
     struct linkSet newSet = {NULL, NULL, NULL};
 
+    if (canUseAlternativeAdminDir()) {
+        stateDir = "/etc/alternatives/admindir";
```

Comment:
I wouldn't use /etc/alternatives/ directory. Prpbably nobody has an alternative named "admindir", but we can't be sure.

Reply:
Race condition: I was trying to debug the CI failures.

The idea is to to move it to `/etc/alternatives.admindir`